doc/circuitikzmanual.tex

```diff
@@ -1106,29 +997,123 @@ \subsection{Resistive bipoles}
 \begin{groupdesc}
     \circuitdescbip*{varistor}{Varistor}{}
     \circuitdescbip*[photoresistor]{phR}{Photoresistor}{photoresistor}
-    \circuitdescbip{thermocouple}{Thermocouple}{}
     \circuitdescbip*[thermistor]{thR}{Thermistor}{thermistor}
     \circuitdescbip*[thermistorptc]{thRp}{PTC thermistor}{thermistor ptc}
     \circuitdescbip*[thermistorntc]{thRn}{NTC thermistor}{thermistor ntc}
-    \circuitdescbip*{fuse}{Fuse}{}
-    \circuitdescbip*{afuse}{Asymmetric fuse}{asymmetric fuse}
 \end{groupdesc}
 
 \subsubsection{Generic sensors anchors}
-Generic sensors have an extra label to help positioning the type of dependence, if needed:
+Generic sensors have an extra anchor named \texttt{label}  to help positioning the type of dependence, if needed:
+
+\begin{LTXexample}[varwidth=true]
+\begin{circuitikz}
+   \draw (0,2) to[sR, l=$R$, name=mySR] ++(3,0);
+   \node [font=\tiny, right] at(mySR.label) {-t\si{\degree}};
+   \draw (0,0) to[sL, l=$L$, name=mySL] ++(3,0);
+   \node [draw, circle, inner sep=2pt] at(mySL.label) {};
+\end{circuitikz}
+\end{LTXexample}
+
+The anchor is positioned just on the corner of the segmented line crossing the component.
+
+\subsubsection{Resistive components customization}
+
+You can change the scale of these components (all the resistive bipoles together) by setting the key \texttt{resistors/scale}  to something different from the default \texttt{1.0}.
```

Comment:
```suggestion
You can change the scale of these components (all the resistive bipoles together) by setting the key \texttt{resistors/scale} (default \texttt{1.0}).  Similarly, you can change the widths by setting \texttt{resistors/width} (default \texttt{1.0}).

Reply:
This along with https://github.com/circuitikz/circuitikz/pull/255/files/3c0f9d6c0c61baac8e22e4f9c83dc1eea498157c#r305571910 should work.